README.md

```diff
@@ -1,59 +1,124 @@
-- Share events from parks to Google calendars
-- Allow users to add comments to both events and parks, as well as add ratings to parks and their facilities
+- [x] Incorporate special feature and facilities information to parks by using additional Vancouver Open Data Portal APIs
+- [x] Filter parks by fields such as rating, park size, facilities, special features
+- [x] Share events from parks to Google calendars
+- [x] Find parks by address
+- [x] Use OAuth-based Google Sign-in to allow users to create an account and save events and parks to their account
+- [x] Allows users to add events to their calendar application (Apple, Google etc)
+- [x] Allow both non-logged-in users and users to add comments to both events and parks, as well as add ratings to parks
 
 **2-3 stretch requirements (plan to complete 1!)**
-- Make the app only useable with an account, make the account secure (
-- Allow users to invite other users within the app
-- Create a cron job that re-populates the park database weekly or monthly to ensure that all the park information is up to date
-- Include “Weekend playfield status”: display events at a playfield so that events are not scheduled at the same time
+- [x] Make the application mobile-friendly (responsive)
+- [] Allow users to invite other users within the app
+- [] Make the app only useable with an account, make the account secure (NOTE: we decided to allow both users with accounts and users without accounts)
+- [] Create a cron job that re-populates the park database weekly or monthly to ensure that all the park information is up to date
 
 **Pick 2 of your minimal requirements and break each of them down into ~2-5 smaller tasks!**
 - Create tables in our own database to represent a park and an event
 - Create a script to get all of the parks data from the Vancouver parks API and then write them into our database (so we don’t have to do it by hand)
 
+## Course technologies used
+
+All the technologies that we learned in this course are utilized in our project:
+
+**HTML/CSS/JavaScript**
+
+We utilized HTML/CSS throughout our application (with React) for displaying/styling components, views.
+JavaScript was used in the frontend and backend to write methods such as for display logic, manipulate objects etc.
+
+**React/Redux** 
+
+React was used to create all the components that we used in our application. We used Redux Toolkit for state management.
+  
+**Node.js/Express.js**
+
+We used these two for building the backend. Our frontend interacts with 
+the endpoints and the backend interacts with the database. Database operations (GET/POST/DELETE/PATCH) are supported 
+depending on the collection.
+
+**MongoDB** 
+
+We included Mongoose as well. We used MongoDB Atlas as our backend cloud database storage of information. 
+We had collections including parks, users, events, park comments, and event comments.
```

Comment:
Do we want to mention facilities and features too?

Reply:
I didn't want to list every single collection, these are the most important